src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  margin: 0;
+  height: 60px;
+  border: 1px solid rgb(175, 174, 174);
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav--link.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.logo {
+  padding: auto;
```

Comment:
The `padding: auto;` is not a valid value for padding. Padding should be specified with length values (e.g., px, em) or percentages. Consider removing or replacing it with a valid value.